.github/workflows/run-target-test-suite.yml

```diff
@@ -0,0 +1,85 @@
+# This workflow runs a specific regression test suite, using the input image
+# name and tag from the workflow call.
+#
+# Required settings in the repository:
+# Environments (exactly the following strings):
+# - production
+# - uat
+# - sit
+#
+# Secrets in each environment:
+# - EDL_USER
+# - EDL_PASSWORD
+#
+# Variables in each environment:
+# - HARMONY_HOST_URL
+#
+# Note: A workflow_call event can only have inputs of type boolean, number or
+# string.
+name: Run test suite
+
+on:
+  workflow_call:
+    inputs:
+      docker-image-name:
+        required: true
+        type: string
```

Comment:
It's a little frustrating that these inputs can't be a "choice" like the `workflow_dispatch` event (manual invocation). Perhaps there's some level of validation that could be added to any parent workflow.